ts/screens/onboarding/OnboardingCompletedScreen.tsx

```diff
@@ -7,8 +7,8 @@ import { InfoScreenComponent } from "../../components/infoScreen/InfoScreenCompo
 import FooterWithButtons from "../../components/ui/FooterWithButtons";
 import { BlockButtonProps } from "../../components/ui/BlockButtons";
 import I18n from "../../i18n";
-import Completed from "../../../img/pictograms/payment-completed.svg";
 import { completeOnboarding } from "../../store/actions/onboarding";
+import PictogramFireworks from "../../components/core/pictograms/svg/PictogramFireworks";
```

Comment:
The correct way to import the `Pictogram` component is:
```suggestion
import { Pictogram } from "../../components/core/pictograms";
```

Reply:
do it